ts/src/index.ts

```diff
@@ -152,7 +168,21 @@ function logError(msg: string, config: Config) {
  * profiles.
  */
 export async function startLocal(config: Config = {}): Promise<void> {
+  // start the v8 heap profiler if heap profiling is enabled. This must be done
+  // before any asynchronous calls are made so samples from the first tick can
+  // be captured.
+  if (!(config.disableHeap === false)) {
```

Comment:
Can it be just `if (!!config.disableHeap)`?

Reply:
Done.